### ISSUE TITLE
Accessibility add link name

### ISSUE THREAD

**Issue by Arhell** [karmada-io/website#25]
/assign 
![Screenshot_1](https://user-images.githubusercontent.com/26163841/159862076-22635643-1016-4468-8181-97121005e44a.png)

An accessible name is a word or phrase coded in a way that assistive technologies can associate it with a specific user interface object. Assistive technologies can then refer to the object by name, not just by type. When a link doesn’t have an accessible name, people who use assistive technologies have no way of knowing its purpose.
